power-apps/responsive-hero-cards-gallery/README.md

```diff
@@ -22,9 +22,9 @@ This is a Power Apps YAML snippet that shows a responsive gallery of clickable h
 
 1. Inside the **Tree View**, select the *screen* or *parent container* you would like to add the snippet to.
 
-1. **Right click** - **Paste** - **Paste code**.
+2. **Right click** - **Paste**
```

Comment:
Please leave the ordered list as 1., 1., 1... This will make sure it keeps on counting from 1 to 2 to 3 automatically and doesn't give us the headache of changing a bunch of steps when a step gets removed or added.